Sources/Sentry/SentryReachability.m

```diff
@@ -155,6 +156,15 @@ + (void)initialize
     }
 }
 
+- (instancetype)init
+{
+    if (self = [super init]) {
+        self.skipRegisteringActualCallbacks = NO;
+    }
+
+    return self;
+}
```

Comment:
m: can we use `#if TEST` instead of this to avoid the extra bytes and processing during release?